docs/install_guides/installing-openvino-from-archive-macos.md

```diff
@@ -41,49 +41,51 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 
 ### <a name="install-core"></a>Step 1: Install OpenVINO Core Components
 
-1. Select and download the OpenVINO™ archive files from [Intel® Distribution of OpenVINO™ toolkit for macOS download page](https://software.intel.com/en-us/openvino-toolkit/choose-download/free-download-macos). There are typically two files for you to download: 
-   ```sh
-   m_openvino_toolkit_<operating system>_<release version>_<package ID>_x86_64.tgz
-   m_openvino_toolkit_<operating system>_<release version>_<package ID>_x86_64.tgz.sha256
-   ``` 
-   where the `.sha256` file is used to verify the success of the download process.
-
-2. Locate the downloaded files in your system. This document assumes the files are in your `Downloads` directory. 
+1. Open a command prompt terminal window. Create a folder for OpenVINO and move into it by issuing the following commands. If the `/opt/intel` folder already exists, skip the `mkdir` command.
```

Comment:
1. Open a command prompt terminal window. You can use the keyboard shortcut: Ctrl+Alt+T

2. Create the `/opt/intel` folder for OpenVINO and switch to it by using the following commands. If the folder already exists, skip the `mkdir` command.
```